Tools/BuildAndPackage.ps1

```diff
@@ -64,35 +64,42 @@ function CleanUp-PrivateDataAndExportVariables($moduleManifest) {
             $fieldNeededFromPSData = @("Tags", "LicenseUri", "ProjectUri", "IconUri", "ReleaseNotes")
             foreach ($field in $fieldNeededFromPSData) {
                 if ($psData.ContainsKey($field)) {
-                    $moduleManifest[$field] = $psData[$field]
+                    # For the array of tags, we have to explicitly convert them.
+                    if ($field -eq "Tags" -and $null -ne $psData[$field]) {
```

Comment:
I don't think that null check does anything. `[string[]]$null` is still `$null`.